ts/context.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         self._metrics = None
         self._limit_max_image_pixels = True
         self.metrics = metrics
+        self.cl_socket = None
```

Comment:
could we add a brief comment around how this is referring to a client socket? Had to spend a minute to figure out what this is refering to